tools/docker-compose/ansible/roles/sources/templates/docker-compose.yml.j2

```diff
@@ -37,6 +37,7 @@ services:
     working_dir: "/awx_devel"
     volumes:
       - "../../../:/awx_devel"
+      - "../../docker-compose/bootstrap_development.sh:/usr/bin/bootstrap_development.sh"
```

Comment:
probably want to remove this, since it seems like a temporary hack to test before building images

Reply:
haha I was trying to sneak it in. okay I'll remove it :)